.env.example

```
@@ -1,14 +1,99 @@
+#
+# CHAIN_ETHEREUM_RPC_URL=wss://...
+# CHAIN_ETHEREUM_CHAIN_ID=1
+# CHAIN_ETHEREUM_SUBGRAPH_URL=https://...
+# CHAIN_ETHEREUM_BACKFILL_START_BLOCK=0
+#
+# Fireblocks maps chain_id -> asset id for every configured chain (see below).
+# Startup fails if a live tokenized asset's network has no registry entry.
+
+# Legacy Base chain (single-chain / Base-only deployments)
 RPC_URL=${RPC_URL}
+CHAIN_ID=8453
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAIN_ID key should go before the RPC_URL key

(UnorderedKey)